gather_istio.sh

```diff
@@ -125,6 +125,7 @@ function getEnvoyConfigForPodsInNamespace() {
   pilotName=$(getPilotName "${controlPlaneNamespace}")
   local podNamespace="${2}"
 
+  echo
```

Comment:
This is just to add an empty line?

Reply:
Yup. The output had empty lines in most places, but not in all.